app/src/components/dialog/YesNoDialog.tsx

```diff
@@ -1,4 +1,4 @@
-import Button, { ButtonProps } from '@mui/material/Button';
+import LoadingButton, { LoadingButtonProps } from '@mui/lab/LoadingButton';
```

Comment:
would it make sense to at this point to remove the old `LoadingButton` component and replace it with the mui one?

Reply:
The `LoadingButton` component you're referring to is actually in SIMS, not in this repo. We can't remove the SIMS `LoadingButton` component until we update MUI to version 5.

Reply:
🤡